frontend/src/scenes/session-recordings/player/controller/PlayerController.tsx

```diff
@@ -37,68 +38,60 @@ export function PlayerController(): JSX.Element {
-                                                onClick={() => {
-                                                    setSpeed(speedToggle)
-                                                }}
-                                            >
-                                                {speedToggle}x
-                                            </LemonButton>
-                                        ))}
-                                    </div>
-                                ),
-                                closeOnClickInside: true,
-                            }}
-                            sideIcon={null}
+            <div className="flex justify-between h-8 gap-2 m-2 mt-1">
+                <div className="flex divide-x gap-2">
+                    <Timestamp />
+                    <div className="flex pl-2 gap-1">
+                        <LemonButton
                             size="small"
+                            onClick={togglePlayPause}
+                            tooltip={
+                                <>
+                                    {showPause ? 'Pause' : 'Play'}
+                                    <KeyboardShortcut space />
+                                </>
+                            }
                         >
-                            {speed}x
-                        </LemonButtonWithDropdown>
-                    </Tooltip>
-
-                    <Tooltip title={`Skip inactivity (${skipInactivitySetting ? 'on' : 'off'})`}>
+                            {showPause ? <IconPause className="text-2xl" /> : <IconPlay className="text-2xl" />}
+                        </LemonButton>
+                        <SeekSkip direction="backward" />
+                        <SeekSkip direction="forward" />
+                        <LemonMenu
+                            data-attr="session-recording-speed-select"
+                            items={PLAYBACK_SPEEDS.map((speedToggle) => ({
+                                label: `${speedToggle}x`,
+                                onClick: () => setSpeed(speedToggle),
+                            }))}
+                        >
+                            <LemonButton size="small" tooltip="Playback speed" sideIcon={null}>
+                                {speed}x
+                            </LemonButton>
+                        </LemonMenu>
+                    </div>
+                    <div className="flex pl-2">
                         <LemonButton
                             size="small"
                             onClick={() => {
                                 setSkipInactivitySetting(!skipInactivitySetting)
                             }}
+                            icon={
+                                <IconSkipInactivity
+                                    className={clsx(
+                                        'text-2xl',
+                                        skipInactivitySetting ? 'text-primary-3000' : 'text-primary-alt'
```

Comment:
this button was so completely invisible to me, that I'd be tempted to check if anyone uses it and move it away into settings or the kebab menu to reduce the amount of default visible interactions

Reply:
Yeah, I certainly wouldn't be against always skipping inactivity. Will leave it in for this PR and add a `data-attr` so that we can assess usage